src/implementations/twig/components/hero-banner/README.md

```diff
@@ -0,0 +1,46 @@
+# ECL Hero Banner
+
+npm package: `@ecl/twig-component-hero-banner`
+
+```shell
+npm install --save @ecl/twig-component-hero-banner
+```
+
+### Parameters
+
+- **"type"** (string) (default: 'default') Type of banner (can be 'default','image','image-shade','primary')
+- **"title"** (string) (default: '') Title of banner
+- **"image"** (string) (default: '') Image for banner (required for image banner type)
+- **"description"** (string) (default: '') Description of banner
+- **"centered"** (bool) (default: true) Define if banner should be centered
+- [DEPRECATED] **"button"** (associative array) (default: predefined structure) predefined structure for EC Button component
+- **"link"** (associative array) (default: predefined structure) predefined structure for EC Link component
```

Comment:
Since we are removing references to the system let's also remove from it here
```suggestion
- **"link"** (associative array) (default: predefined structure) predefined structure for the Link component
```